pnpm-workspace.yaml

```diff
@@ -21,6 +21,7 @@ overrides:
 
 patchedDependencies:
   'density-clustering@1.3.0': 'patches/density-clustering@1.3.0.patch'
+  'domino@2.1.6': 'patches/domino@2.1.6.patch'
```

Comment:
The idea is to use the pnpm patch and remove the qwik dom package from the codebase. This will clean up things.
I did a diff between the current domino lib code and our custom   qwik dom code, the patch is the result of it.